autoblocks/_impl/tracer.py

```diff
@@ -101,6 +123,70 @@ def start_span(self):
                 props["span_id"] = prev_span_id
             self.set_properties(props)
 
+    async def _evaluate_event(self, event: TracerEvent, evaluator: BaseEventEvaluator) -> Optional[Evaluation]:
+        """
+        Evaluates an event using a provided evaluator.
+        """
+        evaluation = None
+        if inspect.iscoroutinefunction(evaluator.evaluate_event):
+            try:
+                evaluation = await evaluator.evaluate_event(event=event)
+            except Exception as err:
+                return Evaluation(
+                    score=0,
+                    metadata={
+                        "error": "Evaluation raised an exception during execution",
+                        "errorDetail": f"{repr(err)}",
+                    },
```

Comment:
I updated the implementation so that evaluators that fail to run will still report a score of 0 with some metadata indicating it was an error. This way it will be obvious is evaluations are failing to run when inspecting traces

Reply:
i would punt on this and continue logging errors, i think we need to come up with a better design and handle them in a first class way